### PR TITLE
Increase Code Coverage

### PR DESCRIPTION
**Pull request by addisonElliott**
Went through and added tests to uncovered routes according to codecov. Went from 88% to 90%!

Small other changes made along the way.

### CODE REVIEW

File: nrrd/reader.py [hunk 1]
```diff
@@ -78,8 +78,8 @@ def _get_field_type(field, custom_field_map):
         return 'double list'
     elif field in ['kinds', 'labels', 'units', 'space units', 'centerings']:
         return 'string list'
-    elif field in []:
-        return 'int vector'
+    # elif field in []:
+    #     return 'int vector'
     elif field in ['space origin']:
         return 'double vector'
     elif field in ['measurement frame']:
```
Comment (tashrifbillah) [inline nrrd/reader.py]: Isn't this the place where Isaiah and I corrected 'double matrix'? Not sure if you are working on a different branch.

File: nrrd/tests/test_reading.py [hunk 2]
```diff
@@ -362,6 +362,41 @@ def test_detached_header_no_filename(self):
...
                 nrrd.read_data(header, fh)
 
+    def test_invalid_lineskip(self):
+        with open(RAW_NRRD_FILE_PATH, 'rb') as fh:
+            header = nrrd.read_header(fh)
+            np.testing.assert_equal(self.expected_header, header)
+
+            # Set the line skip to be incorrect
+            header['line skip'] = -1
+
```
Comment (tashrifbillah) [inline nrrd/tests/test_reading.py]: I thought I added this one before, but thanks if I haven't.

**Comment by tashrifbillah** [comment]
Don't see any obvious discrepancy. I would give it a green.

File: nrrd/reader.py (hunk 1 above)
Comment (addisonElliott) [reply nrrd/reader.py]: Yeah, this is the place. I just checked this PR and it's up to date. The `measurement frame` field type is 'double matrix' as corrected.

File: nrrd/tests/test_reading.py (hunk 2 above)
Comment (addisonElliott) [reply nrrd/tests/test_reading.py]: I think you tested the `byte skip` and `line skip` for all encoding except raw. But that's one benefit of code coverage is for, telling us when something isn't tested.

**Comment by addisonElliott** [comment]
@tashrifbillah Is this how long you meant by you needed time? That's nothing! I was expecting on the order of days to weeks!

Thank you for the review.

**Pull request merged**